sailthru/Sailthru_Client.php

```diff
@@ -1434,9 +1434,22 @@ protected function httpRequestCurl($action, array $data, $method = 'POST', $opti
         curl_setopt($ch, CURLOPT_HTTPHEADER, $this->httpHeaders);
         $response = curl_exec($ch);
         $this->lastResponseInfo = curl_getinfo($ch);
+
+        // Get the curl error message if there is one
+        $errorMessage = false === $response ? curl_error($ch) : null;
+
         curl_close($ch);
 
+        if (false === $response) {
+            // There's a curl error! throw an exception which gives us some details
+            throw new Sailthru_Client_Exception(
+                sprintf('Error with curl transport: %s', $errorMessage),
```

Comment:
1) Can you change to string interpolation? It's used more frequently throughout the library
2) Can you reduce to "curl error: <message>"
e.g. `"curl error: {$errorMessage}"`